stable/phpbb/values.yaml

```diff
@@ -32,15 +36,49 @@ phpbbEmail: user@example.com
 # smtpUser:
 # smtpPassword:
 
+##
+## External database configuration
+##
+externalDatabase:
+  ## Database host
+  # host:
+
+  ## Database user
+  # user: bn_mediawiki
+
+  ## Database password
+  # password:
+
+  ## Database name
+  # database: bitnami_mediawiki
+
 ##
 ## MariaDB chart configuration
 ##
 mariadb:
+  ## Whether to use the database specified as a requirement or not. For example, to configure the chart with an existing database server.
```

Comment:
How about changing this to..

"## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to `false` and configure the `externalDatabase` parameters"

Reply:
Yes, much better